src/patternfly/components/Nav/nav.scss

```diff
@@ -516,6 +554,41 @@
       border-bottom: var(--pf-c-nav__item--before--BorderWidth) solid var(--pf-c-nav__item--before--BorderColor);
     }
   }
+
+  &.pf-m-flyout {
+    --pf-c-nav__item--MarginTop: var(--pf-c-nav__item--m-flyout--MarginTop);
+
+    &:hover {
+      background-color: var(--pf-c-nav__item--m-flyout--hover--BackgroundColor);
+    }
+
+    &:focus {
+      background-color: var(--pf-c-nav__item--m-flyout--focus--BackgroundColor);
+    }
+
+    &:active {
+      background-color: var(--pf-c-nav__item--m-flyout--active--BackgroundColor);
+    }
+
+    .pf-c-nav__subnav {
+      --pf-c-nav__link--PaddingTop: var(--pf-c-nav__item--m-flyout__subnav__link--PaddingTop);
+      --pf-c-nav__link--PaddingBottom: var(--pf-c-nav__item--m-flyout__subnav__link--PaddingBottom);
+      --pf-c-nav__subnav--PaddingBottom: var(--pf-c-nav__item--m-flyout--subnav--PaddingBottom);
+      --pf-c-nav__subnav--PaddingLeft: var(--pf-c-nav__item--m-flyout--subnav--PaddingLeft);
+      --pf-c-nav__subnav--MaxHeight: var(--pf-c-nav__item--m-flyout__subnav--MaxHeight);
+      --pf-c-nav__link--FontSize: var(--pf-c-nav__item--m-flyout__subnav__link--FontSize);
+      --pf-c-nav__link--before--BorderBottomWidth: var(--pf-c-nav__item--m-flyout__subnav__link--before--BorderBottomWidth);
+      --pf-c-nav__link--before--BorderColor: var(--pf-c-nav__item--m-flyout__subnav__link--before--BorderColor);
+
+      position: absolute;
+      top: var(--pf-c-nav__item--m-flyout__subnav--Top);
+      left: var(--pf-c-nav__item--m-flyout__subnav--Left);
+      z-index: calc(var(--pf-c-nav__item--m-flyout__subnav--ZIndex) + 1);
```

Comment:
The subnavs appear to work fine without a `z-index` - is that necessary? If so I'm curious about the `+1`, and if that's needed, should it exist in `--pf-c-nav__item--m-flyout__subnav--ZIndex`?